includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -94,8 +94,8 @@ public function read( &$coupon ) {
 		$coupon->set_props( array(
 			'code'                        => $post_object->post_title,
 			'description'                 => $post_object->post_excerpt,
-			'date_created'                => strtotime( $post_object->post_date_gmt ),
-			'date_modified'               => strtotime( $post_object->post_modified_gmt ),
+			'date_created'                => 0 < $post_object->post_date_gmt ? wc_string_to_timestamp( $post_object->post_date_gmt ) : null,
```

Comment:
Auto-drafts have a `post_date_gmt` of `0000-00-00 00:00:00`, so this will evaluate to true as that string is greater than 0. What is this check supposed to catch?